pennylane/transforms/decompositions/gridsynth.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Alias transform function for the Ross-Selinger decomposition (GridSynth) for qjit + capture."""
+
+from functools import partial
+
+from pennylane.transforms.core import transform
+
+
+@partial(transform, pass_name="gridsynth")
+def gridsynth(tape, *, epsilon, ppr_basis):
+    r"""Decomposes RZ and PhaseShift gates into Clifford+T basis or PPR basis.
+
+    .. warning::
+
+        This transform requires QJIT and capture to be enabled,
+        as it is a wrapper for Catalyst's ``gridsynth`` compilation pass.
+
+    Args:
+        tape (QNode): A quantum circuit.
+        epsilon (float): The maximum error for the discretization.
```

Comment:
Should this have a default value? Let's also update the description such that it aligns more with the existing `clifford_t_decomposition`.
```suggestion
        epsilon (float): The maximum permissible operator norm error per rotation gate.
```

Reply:
You can just mention that users should consult the Catalyst documentation for further information.
You can say this in the warning about capture and qjit.